app/models/manageiq/providers/azure/cloud_manager/event_catcher.rb

```diff
@@ -5,10 +5,6 @@ class ManageIQ::Providers::Azure::CloudManager::EventCatcher < ManageIQ::Provide
   # providers that do not support it.
   #
   def self.all_valid_ems_in_zone
-    super.select do |ems|
-      ems.supports?(:timeline).tap do |available|
-        _log.info(ems.unsupported_reason(:timeline) + " [#{ems.provider_region}]") unless available
```

Comment:
`all_valid_ems_in_zone` is called quite often by the `sync_workers` loop in MiqServer which means this gets logged very frequently if any EMSs don't support event capture.

I think we need a better way to bubble up why events can't be collected for a provider to the user than via the logs.

E.g. we could use the supports features to show this info on the UI on the provider dashboard.